### DISCUSSION TITLE
Additional Page Control Event Triggers

### DISCUSSION THREAD

**Discussion by xHorizon96** [microsoft/ALAppExtensions#16604] [Ideas] [answered]
Hi,
would it be possible to add additional events on page controls, that we can subscribe to?
For example:
- OnEnter (Like in WinForms on TextBoxes)
- OnLeave (Also like in WinForms on TextBoxes)

Sometimes there is a need to make some calculations or to trigger code on entering certain controls on pages without using a dedicated action trigger just for that case or using other page triggers.

**Comment by JesperSchulz** [comment, marked as answer]
That's a good suggestion, but not really related to this repository. It's more of a request to our platform. Maybe try creating an idea on aka.ms/bcideas.
